tdigest/merging_digest.go

```diff
@@ -388,6 +379,19 @@ func (td *MergingDigest) Merge(other *MergingDigest) {
 	td.reciprocalSum = oldReciprocalSum + other.reciprocalSum
 }
 
+// Merge another digest data into this one. Neither td nor other can be shared
+// concurrently during the execution of this method.
+func (td *MergingDigest) MergeData(other *MergingDigestData) {
```

Comment:
No uses of this in here, is it for something else? Tests?

Reply:
Needs a test. But yeah, just seemed terrible to instantiate a new merging digest just to merge if i had the data serialized.